modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1428,6 +1431,83 @@ public void testAddObjectEntryWithAttachmentObjectField() throws Exception {
 			objectDefinition.getObjectDefinitionId());
 	}
 
+	@Test
+	public void testAddObjectEntryWithPicklistObjectField() throws Exception {
+		ListTypeDefinition listTypeDefinition =
+			listTypeDefinitionLocalService.addListTypeDefinition(
+				null, TestPropsValues.getUserId(),
+				Collections.singletonMap(
+					LocaleUtil.getDefault(), RandomTestUtil.randomString()),
+				false,
+				ListUtil.fromArray(
+					ListTypeEntryUtil.createListTypeEntry(
+						"listTypeEntryKey1",
+						Collections.singletonMap(
+							LocaleUtil.US, RandomTestUtil.randomString())),
+					ListTypeEntryUtil.createListTypeEntry(
+						"listTypeEntryKey2",
+						Collections.singletonMap(
+							LocaleUtil.US, RandomTestUtil.randomString()))));
+
+		ObjectDefinition objectDefinition = _createObjectDefinition(
+			Collections.singletonList(
+				new PicklistObjectFieldBuilder(
+				).indexed(
+					true
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).listTypeDefinitionId(
+					listTypeDefinition.getListTypeDefinitionId()
+				).name(
+					"picklistObjectFieldName"
+				).build()));
+
+		try {
+			_assertObjectEntryProperties(null, null, objectDefinition);
+			_assertObjectEntryProperties(
+				"listTypeEntryKey1", "listTypeEntryKey1", objectDefinition);
+			_assertObjectEntryProperties(
+				"listTypeEntryKey2", "listTypeEntryKey2", objectDefinition);
+
+			long objectEntryId = _assertObjectEntryProperties(
+				StringPool.BLANK, StringPool.BLANK, objectDefinition);
+
+			ObjectField objectField = _objectFieldLocalService.fetchObjectField(
+				objectDefinition.getObjectDefinitionId(),
+				"picklistObjectFieldName");
+
+			_objectFieldSettingLocalService.addObjectFieldSetting(
+				TestPropsValues.getUserId(), objectField.getObjectFieldId(),
+				ObjectFieldSettingConstants.NAME_DEFAULT_VALUE,
+				"listTypeEntryKey1");
+			_objectFieldSettingLocalService.addObjectFieldSetting(
+				TestPropsValues.getUserId(), objectField.getObjectFieldId(),
+				ObjectFieldSettingConstants.NAME_DEFAULT_VALUE_TYPE,
+				ObjectFieldSettingConstants.VALUE_INPUT_AS_VALUE);
+
+			Assert.assertEquals(
+				StringPool.BLANK,
+				_getListEntryKey(
+					_defaultObjectEntryManager.getObjectEntry(
+						_simpleDTOConverterContext, objectDefinition,
+						objectEntryId)));
```

Comment:
I know that the ticket is related to GET, but we could also test the case in which we try to update an object entry (empty value for picklist) to verify if the default value will not be set as value("") and with a null value.
What do you think?